templates/dex.yaml

```diff
@@ -21,6 +21,8 @@ spec:
   requires:
     matchLabels:
       kubeaddons.mesosphere.io/provides: ingresscontroller
+      kubeaddons.mesosphere.io/name: cert-manager
+      values.chart.helm.kubeaddons.mesosphere.io/cert-manager-setup: "https://raw.githubusercontent.com/mesosphere/charts/6c43b8ab10108fb1adba5c6dd10e800e5f1abdd0/staging/cert-manager-setup/values.yaml"
```

Comment:
shouldn't need this line, the name label should be sufficient.

Reply:
I'll remove it